### PR TITLE
Clarified the README file

### PR DESCRIPTION
**Pull request by NikNikovsky**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -16,7 +16,7 @@ If you want to read about why this fork was created, check out [our FAQ page](ht
 
 ## Development Builds
 
-There are per-commit development builds available [here](https://github.com/PolyMC/PolyMC/actions). These have debug information in the binaries, so their file sizes are relatively larger.
+There are per-commit development builds available [here](https://github.com/PolyMC/PolyMC/actions). These have to debug information in the binaries, so their file sizes are relatively larger.
```
Comment (LennyMcLennington) [inline README.md]: This is just incorrect

File: README.md [hunk 2]
```diff
@@ -26,7 +26,7 @@ For flatpak, you can use [flathub-beta](https://discourse.flathub.org/t/how-to-u
 
 # Help & Support
 
-Feel free to create an issue if you need help. However, you might find it easier to ask in the Discord server.
+Feel free to create an issue if you need help. However, you might find it easier to ask on the Discord server.
```
Comment (LennyMcLennington) [inline README.md]: This is just incorrect

File: README.md [hunk 3]
```diff
@@ -3,7 +3,7 @@
 <img src="./program_info/polymc-header.svg#gh-dark-mode-only" alt="PolyMC logo" width="50%"/>
 </p>
 
-PolyMC is a custom launcher for Minecraft that focuses on predictability, long term stability and simplicity.
+PolyMC is a custom launcher for Minecraft that focuses on predictability, long-term stability, and simplicity.
```
Comment (LennyMcLennington) [inline README.md:6]: This is fine

**Review by LennyMcLennington — CHANGES_REQUESTED** [review]
Most of these changes just make it grammatically or factually incorrect.

File: README.md (hunk 1 above)
Comment (NikNikovsky) [reply README.md]: Will fix

File: README.md (hunk 2 above)
Comment (NikNikovsky) [reply README.md]: Will fix

File: README.md (hunk 1 above)
Comment (NikNikovsky) [reply README.md]: Fixed

File: README.md (hunk 2 above)
Comment (NikNikovsky) [reply README.md]: Fixed

File: README.md (hunk 3 above)
Comment (NikNikovsky) [reply README.md:6]: If it's fine, why bother fixing

**Comment by NikNikovsky** [comment]
I didn't have time to do it.
(I am in the GMT+1 timezone)

**Pull request closed without merging**